docs/arcadia-users-group/20240723-design-fundamentals/lesson.md

```diff
@@ -0,0 +1,240 @@
+# Fundamentals of Design
+
+This AUG lesson is a brief introduction to the principles of design, with a focus on how they can be applied to scientific visualizations.  We will cover the following topics:
+- Why we should care about good design
+- **Layout:** how we arrange and compose figures to guide the viewer
+  - Gestalt principles
+  - Visual hierarchy
+  - Composition
+  - Grids, grouping, and alignment
+- Layout exercise
+- **Design details:** how to choose visual elements to communicate better
+  - Color
+  - Typography and hierachy
+  - Symbols
+- Design critique
+
+
+## Introduction
+### Where we're starting from
+
+> “Beauty is a very important entry point for readers to get interested about the visualization and be willing to explore more.  Beauty cannot replace functionality, but beauty and functionality together achieve more.” – *Better Data Visualization*
+
+Beautiful scientific visualizations are able to grab your attention. But design is more than just aesthetics – it is a critical component of effective communication.  Efficient design is functional; it distills complex information into simple and clear contrasts, patterns, or sequences.
+
+In this lesson, we’re going to discuss design principles as they emerged as a formalized system, most famously established by German/Swiss design movements in the early 20th century.  These are time-tested tools which draw from graphical history/culture and perception science.
+
+> “Traditionally, graphic design theory has privileged intuition and creativity over empirical research…. [But] it can be argued that the art-based principles of graphic design - including (but not limited to ) contrast, hierarchy, repetition, alignment, and color - are in fact theories proven through a long history of successful experimentation in practice.  Indeed, graphic designers - through professional practice - have tested and restested to the point where it makes sense to refer to these theories as laws or principles.” – *Audrey G. Bennett, Design Studies: Theory and Research in Graphic Design*
+
+<details>
+    <summary> A note about decolonizing design </summary>
+    There’s been a lot of discussion of and work towards decolonizing design, and I can speak to some of the projects I’m familiar with.  But I also feel strongly that the visual systems codified by Bauhaus and Swiss design are basically summaries of very ancient systems of visual cognition that don’t belong to Europe, they just had the power/structure to formalize and disseminate it. (Ex: the golden ratio concept (often associated with Swiss grid design) is at least as old as ancient Greece and there’s evidence of stronger ties to ancient African civilizations.) Not to mention the impact of millenia of trade and travel and centuries of colonization that had incalculable influence on European thought.
```

Comment:
We should avoid using "I" throughout; this document is meant to reflect the training perspective of the whole organization, not an individual.